src/parser.rs:7:5 - error[E0432]: unresolved import `rug::ops::NegAssign`

See https://travis-ci.com/github/mthom/scryer-prolog/jobs/315547668
```
 Compiling prolog_parser v0.8.49
     Running `rustc --crate-name prolog_parser /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/prolog_parser-0.8.49/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type lib --emit=dep-info,metadata,link -C debuginfo=2 --cfg 'feature="num"' --cfg 'feature="num-rug-adapter"' -C metadata=07e9b25620cc8d14 -C extra-filename=-07e9b25620cc8d14 --out-dir /home/travis/build/mthom/scryer-prolog/target/debug/deps -L dependency=/home/travis/build/mthom/scryer-prolog/target/debug/deps --extern lexical=/home/travis/build/mthom/scryer-prolog/target/debug/deps/liblexical-c8d36ff4339b7273.rmeta --extern num_rug_adapter=/home/travis/build/mthom/scryer-prolog/target/debug/deps/libnum_rug_adapter-a91ca42a72ee10bc.rmeta --extern ordered_float=/home/travis/build/mthom/scryer-prolog/target/debug/deps/libordered_float-04e0015dbc4456cf.rmeta --extern unicode_reader=/home/travis/build/mthom/scryer-prolog/target/debug/deps/libunicode_reader-97a6913793916f3e.rmeta --cap-lints allow`
error[E0432]: unresolved import `rug::ops::NegAssign`
 --> /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/prolog_parser-0.8.49/src/parser.rs:7:5
  |
7 | use rug::ops::NegAssign;
  |     ^^^^^^^^^^---------
  |     |         |
  |     |         help: a similar name exists in the module: `PowAssign`
  |     no `NegAssign` in `ops`
error: aborting due to previous error
For more information about this error, try `rustc --explain E0432`.
error: could not compile `prolog_parser`.
Caused by:
  process didn't exit successfully: `rustc --crate-name prolog_parser /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/prolog_parser-0.8.49/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type lib --emit=dep-info,metadata,link -C debuginfo=2 --cfg 'feature="num"' --cfg 'feature="num-rug-adapter"' -C metadata=07e9b25620cc8d14 -C extra-filename=-07e9b25620cc8d14 --out-dir /home/travis/build/mthom/scryer-prolog/target/debug/deps -L dependency=/home/travis/build/mthom/scryer-prolog/target/debug/deps --extern lexical=/home/travis/build/mthom/scryer-prolog/target/debug/deps/liblexical-c8d36ff4339b7273.rmeta --extern num_rug_adapter=/home/travis/build/mthom/scryer-prolog/target/debug/deps/libnum_rug_adapter-a91ca42a72ee10bc.rmeta --extern ordered_float=/home/travis/build/mthom/scryer-prolog/target/debug/deps/libordered_float-04e0015dbc4456cf.rmeta --extern unicode_reader=/home/travis/build/mthom/scryer-prolog/target/debug/deps/libunicode_reader-97a6913793916f3e.rmeta --cap-lints allow` (exit code: 1)
The command "cargo test --verbose --all --no-default-features --features num" exited with 101.
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -10,7 +10,7 @@ license = "BSD-3-Clause"
 lexical = "2.1.0"
 ordered-float = "0.5.0"
 rug = { optional = true, version = "1.4.0" }
-num-rug-adapter = { optional = true, version = "0.1.1" }
+num-rug-adapter = { optional = true, version = "0.1.2" }
 unicode_reader = "1.0.0"
 
 [lib]

